### ISSUE TITLE
Travis build fails: incompatible with composer-plugin-api[2.0.0]

### ISSUE THREAD

**Issue by SbstnS** [SemanticMediaWiki/SemanticResultFormats#627]
### Issue

After I started the PR #626 to extend Gallery format with the property "captiontemplate"  the travis build fails because composer-plugin-api seems to be not compatible with build setup.

It seems that this is not a problem with my code but with some composer requirements on MW 1.3.x 

> composer.json
```
"composer-plugin-api": "^1.1",
```

**Comment by gesinn-it** [comment]
It seems that the CI is using Composer 2.x

```
$ composer --version
Composer version 2.0.8 2020-12-03 17:20:38
```

which is not yet supported, see https://phabricator.wikimedia.org/T266417

**Comment by gesinn-it** [comment]
closed via #628
